src/main/java/uk/gov/ons/ctp/response/action/scheduled/ReportScheduler.java

```diff
@@ -35,21 +35,23 @@
   @PostConstruct
   public void init() {
     reportDistributedInstanceManager.incrementInstanceCount(DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT);
-    log.info(
-        "{} {} instance/s running",
-        reportDistributedInstanceManager.getInstanceCount(DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT),
-        DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT);
+    log.with(
+            DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT,
+            reportDistributedInstanceManager.getInstanceCount(
+                DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT))
+        .info("instance/s running");
   }
 
   /** Clean up report scheduler on bean destruction */
   @PreDestroy
   public void cleanUp() {
     reportDistributedInstanceManager.decrementInstanceCount(DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT);
     reportDistributedLockManager.unlockInstanceLocks();
-    log.info(
-        "{} {} instance/s running",
-        reportDistributedInstanceManager.getInstanceCount(DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT),
-        DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT);
+    log.with(
+            DISTRIBUTED_OBJECT_KEY_INSTANCE_COUNT,
```

Comment:
the key is snake_case?

Reply:
`DISTRIBUTED_OBJECT_KEY_REPORT_LATCH = "reportlatch"` not quite snake case but this is the actual name given to the instance

Reply:
same as the other one ^^